resources/shuttle-aws-rds.mdx

```diff
@@ -10,6 +10,18 @@ This plugin provisions databases on AWS RDS using [Shuttle](https://www.shuttle.
 - MariaDB
 
 {/* TODO: Write more about the advantages of RDS vs shared */}
+The RDS instance created by Shuttle has the following specifications and features by default:
+- 2 vCPU
+- 1 GB Memory
+- 30 GB Storage
```

Comment:
It's 20 GiB currently: https://github.com/shuttle-hq/shuttle/blob/main/provisioner/src/lib.rs#L290
```suggestion
- 20 GiB Storage
```